src/main/java/teammates/ui/controller/InstructorCommentsPageData.java

```diff
@@ -146,6 +120,31 @@ public boolean isViewingDraft() {
         return isViewingDraft;
     }        
 
+    private String getRecipientNames(Set<String> recipients) {
```

Comment:
Can you add a header comment for this method? It's not clear what exactly the method is supposed to do.